docs/development/module/fields.md

```diff
@@ -140,8 +212,8 @@ dependencies:
   enforced:
     module:
       - my_module
-id: monitor
-label: Monitor
+id: organic
```

Comment:
Great Job,
Perhaps an additional comment on id in file name should match the `id` in the file `farm_flag.flag.[id].yml` ?